### PR TITLE
Update and rename README.rst to README.md

### PR DESCRIPTION
**Pull request by farlin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2069 @@
...
+## License
+
+Apipie-rails is released under the [MIT
+License](https://opensource.org/licenses/MIT)
+
+# Documentation
+
+::: {.contents depth="2"}
+[Table Of Contents]{.title-ref}
+:::
```
Comment (ekohl) [inline README.md:81]: This doesn't work in markdown

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2069 @@
...
+
+# DSL Reference
+
+## Resource Description
```
Comment (ekohl) [inline README.md:85]: This also doesn't render that well

**Review by ekohl — COMMENTED** [review]
I see https://www.rubydoc.info/gems/apipie-rails/1.3.0 doesn't render well and GitHub also has trouble with the RST file, so I'm not oppose to this. I do think it needs a bit more work.

**Review by mathieujobin — COMMENTED** [review]
pending review...

**Comment by ekohl** [comment]
https://github.com/Apipie/apipie-rails/pull/920 was merged.

**Comment by mathieujobin** [comment]
thank you

**Pull request closed without merging**